README.md

```diff
@@ -441,6 +441,25 @@ contact_page:
       </div>
 ```
 
+#### Header
```

Comment:
I think this should be called "Sticky message" and should be listed after "Second logo" above.